textdb/pom.xml

```diff
@@ -21,12 +22,10 @@
             <dependency>
                 <groupId>junit</groupId>
                 <artifactId>junit</artifactId>
-                <version>${junit.version}</version>

```

Comment:
Version information is added in all other pom.xml files, so the main pom.xml doesn't need to declare version.